src/test/java/com/epam/dataservice/PoliceForceServiceTest.java

```diff
@@ -6,19 +6,36 @@
 import org.junit.Test;
 
 public class PoliceForceServiceTest {
-private	static PoliceForceService testService; 
-@BeforeClass
-public static void init(){
-	testService = new PoliceForceService();
-}
+	private static PoliceForceService testService;
+
+	@BeforeClass
+	public static void init() {
+		testService = new PoliceForceService();
+	}
+
+	@Test
+	public void testGetContactNo_valid_case1() {
```

Comment:
again numbers in tests names should be avoided. And try to follow one approach - camel case or underscore (_).  It looks a bit messy when someone mixing it together ;)